map/framework.cpp

```diff
@@ -2043,6 +2043,32 @@ void Framework::DeactivateHotelSearchMark()
 void Framework::OnTapEvent(place_page::BuildInfo const & buildInfo)
 {
   auto placePageInfo = BuildPlacePageInfo(buildInfo);
+  bool isRoutePoint = placePageInfo.has_value() && placePageInfo->IsRoutePoint();
+
+  if (m_routingManager.IsRoutingActive() \
+      && m_routingManager.GetCurrentRouterType() == routing::RouterType::Ruler \
+      && !buildInfo.m_isLongTap \
```

Comment:
Remove \, it is not used in c++ (except macroses)

Reply:
Removed backslashes